stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/adapter/viewholder/ChannelViewHolder.kt

```diff
@@ -77,10 +76,15 @@ internal class ChannelViewHolder @JvmOverloads constructor(
                 }
                 root.apply {
                     setOnClickListener {
-                        channelClickListener.onClick(channel)
+                        if (!swiping) {
+                            channelClickListener.onClick(channel)
+                        }
                     }
                     setOnLongClickListener {
-                        channelLongClickListener.onLongClick(channel)
+                        when {
+                            !swiping -> channelLongClickListener.onLongClick(channel)
```

Comment:
An if/else here seams easier to read.

Reply:
Sure, I don't have a preference so I'll change it.